app/build.gradle

```diff
@@ -25,8 +27,8 @@ android {
         applicationId "org.mozilla.fenix"
         minSdkVersion Config.minSdkVersion
         targetSdkVersion Config.targetSdkVersion
-        versionCode Config.versionCode
-        versionName Config.versionName + Config.generateVersionSuffix()
+        versionCode 1
```

Comment:
Nit: do we still want to keep `versionCode 1`. My understanding was that very first APK has already been uploaded to Google Play.

Reply:
You're right that the first APK has already been uploaded to Google Play.
`Config.versionCode` used to just be `1`, so I'm just inlining that value to the one place it's used (here) so it's more obvious at-a-glance.

Note that this is overridden separately [here](https://github.com/mozilla-mobile/fenix/pull/1893/files#diff-39e7d8c00954e920b98e7636f0ac30b2R157) for release builds :)